src/handlers/comment/handlers/table.ts

```diff
@@ -4,21 +4,27 @@ export const tableComment = ({
   multiplier,
   reason,
   bounty,
+  isBountyStale,
+  days,
 }: {
   deadline: string;
   wallet: string;
   multiplier?: string;
   reason?: string;
   bounty?: string;
+  isBountyStale?: boolean | null;
+  days?: number;
 }) => {
   return `
 <code>
-
   <table>
-  <tr>
-    <td></td>
-    <td></td>
-  </tr>
+  ${
+    isBountyStale == null
+      ? null
```

Comment:
`null` will be outputted to the string
why not just use `isBountyStale ? "warning..." : ""`

Reply:
My thinking is that ```isBountyStale``` can only represent that it is or isn't stale but it can't represent both those and inactive at the same time.

I wasn't aware that null would be output as text in the html in the rendered html, I wasn't able to produce this but I will look into it

Reply:
ah I feel you now, if it's false show nothing, if its disabled show nothing so no need to check if null